pkg/action/executor/gitops/github_manager.go

```diff
@@ -464,21 +450,75 @@ func (g *GitHandler) updateRemote(resName, resNamespace, clusterId string, patch
 	return nil, nil
 }
 
-func (g *GitHandler) patchYamlContent(resName, branchName string, patches []PatchItem) (bool, []byte, error) {
-	yamlContent, err := g.getRemoteFileContent(resName, g.path, branchName)
+// We find the diff between the resource in the source of truth and the liveRes using
```

Comment:
Is liveRes a typo?

Reply:
Not really, I meant `live Resource`. I will expand for it to be more correct.